### PR TITLE
Improve aspect on large screens

### PR DESCRIPTION
**Pull request by druc**
For #26

@samuelmeuli I tried increasing the font for larger screens but it ended up looking bad - I like the 16px  version way better. I still believe the paragraph spacing should be at least one line though.

Let me know what you think.

Thanks again for building this. It's really nice to write in it! ❤️

### CODE REVIEW

**Comment by samuelmeuli** [comment]
I like it, definitely an improvement! I'll test the changes on a larger screen tomorrow.

How did you arrive at the values for the media query breakpoints? It might be nice to combine some of them into variables.

File: src/renderer/assets/styles/base/_variables.scss [hunk 1]
```diff
@@ -51,7 +51,8 @@ $spacing-rel-large: 6vw;
 $border-radius-button: 10px;
 $border-radius-input: 5px;
 $box-shadow: 0.5px 0.5px 1px 1px rgba(0, 0, 0, 0.18);
-$box-shadow-inset: inset 0.5px 0.5px 1px 1px rgba(0, 0, 0, 0.13);
+$box-shadow-inset: 0 1px 3px 0 rgba(0, 0, 0, 0.1), 0 1px 2px 0 rgba(0, 0, 0, 0.06);
```
Comment (samuelmeuli) [inline src/renderer/assets/styles/base/_variables.scss]: In my opinion, input fields should have an inset shadow, this makes it clear to the user that they're editable.

File: src/renderer/assets/styles/components/_editor.scss [hunk 2]
```diff
@@ -9,9 +10,14 @@
 	display: grid;
 	grid-template-rows: auto auto 1fr;
 	width: 100%;
+	max-width: $max-editor-width;
```
Comment (samuelmeuli) [inline src/renderer/assets/styles/components/_editor.scss]: This line isn't needed.
```suggestion
```

File: src/renderer/assets/styles/components/_editor.scss [hunk 3]
```diff
@@ -9,9 +10,14 @@
...
 	height: 100%;
-	padding: $spacing-rel-small (2 * $spacing-rel-small) $spacing-rel-small $spacing-rel-small;
+	padding: $spacing-large (2.25 * $spacing-large) $spacing-large $spacing-large;
 	overflow-y: auto;
+
+	@media (min-width: 1224px) {
```
Comment (samuelmeuli) [inline src/renderer/assets/styles/components/_editor.scss]: Let's use the actual size of the editor toolbar for the calculations here:

```scss
// Editor padding (width and margin of editor buttons are added to right padding)
$editor-buttons-width: $calendar-cell-size + $spacing-medium;
@media (max-width: 1224px) {
	padding: $spacing-large ($spacing-large + $editor-buttons-width) $spacing-large $spacing-large;
}
@media (min-width: 1225px) {
	padding: (2 * $spacing-large) (2 * $spacing-large + $editor-buttons-width) (2 * $spacing-large)
		(2 * $spacing-large);
}
```

**Review by samuelmeuli — CHANGES_REQUESTED** [review]

**Comment by samuelmeuli** [comment]
And regarding the paragraph spacing: How about setting `line-height: 1.9` and applying `margin-bottom: 1em` to `.editor-text-wrapper .public-DraftStyleDefault-block` on larger screens?

File: src/renderer/assets/styles/base/_variables.scss (hunk 1 above)
Comment (druc) [reply src/renderer/assets/styles/base/_variables.scss]: Yup. I've only added this so the editor could pop up a bit more. I forgot about the inputs.

**Comment by druc** [comment]
Now that we constrained the editor from going full width I don't see any reason to increase the `line-height` or `margin-bottom` **only** on larger screens.

Whenever I'm undecided about typography I usually use this tool: https://grtcalculator.com/
Even though I don't agree with it every time, it does provide a great start.

I'd go with the 1.5em spacing, but I'm a sucker for large spaces between paragraphs, so I understand other people might having their content more condensed.

Should we go with `line-height: 1.7em`, `margin-bottom: 1em`?

PS: regarding the media query 1224px breakpoint, I just needed something larger than 1200px, so I googled the most used media queries. Apparently 1224px is used for desktops and laptops 🤷‍♂.

10px margin
10px

1em margin
1em

1.5em margin
1 5em

File: src/renderer/assets/styles/components/_editor.scss [hunk 4]
```diff
@@ -10,8 +11,17 @@
 	grid-template-rows: auto auto 1fr;
 	width: 100%;
 	height: 100%;
-	padding: $spacing-rel-small (2 * $spacing-rel-small) $spacing-rel-small $spacing-rel-small;
+	padding: $spacing-large (2.25 * $spacing-large) $spacing-large $spacing-large;
```
Comment (samuelmeuli) [inline src/renderer/assets/styles/components/_editor.scss]:
```suggestion
```

**Review by samuelmeuli — CHANGES_REQUESTED** [review]

**Comment by samuelmeuli** [comment]
That's a cool website, thanks for the tip!

Ok, let's do the line/paragraph spacing like you suggested. Let's use `margin-top: 1em` though and remove the bottom margin from the title to make the spacing consistent.

Also, because of that change, I think we should increase the editor padding on small screens slightly again (maybe to `1.25 * $spacing-large` or what we had before), otherwise the paragraph spacing is larger than the padding, which looks a bit odd to me.

Finally, if you could extract the break points into variables, I'll be happy to merge this afterwards. Sorry for the nitpicking 😄

**Comment by druc** [comment]
- added `margin-top: 1em`
- removed `margin-bottom` from title
- increased padding on small screens
- extracted the media query values into `$screen-size` variables. 
- increased the media query for one-column view. Anything smaller than 1000px looks cramped anyway.

We now have: 1024px for tablets, 1224px for desktops/laptops and 1824px for larger screens.

Don't worry about nitpicking. As long as the app gets better, I don't mind tweaking stuff around.

**Review by samuelmeuli — APPROVED** [review]

**Comment by samuelmeuli** [comment]
Awesome, thank you! :)

**Pull request merged**